recipes/fpng/all/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.15)
+project(fpng LANGUAGES CXX)
+
+include(GNUInstallDirs)
+
+add_library(fpng ${FPNG_SRC_DIR}/src/fpng.cpp)
+set_target_properties(fpng PROPERTIES
+    PUBLIC_HEADER ${FPNG_SRC_DIR}/src/fpng.h
+    WINDOWS_EXPORT_ALL_SYMBOLS ON
```

Comment:
I don't see the project build files doing the equivalent, so the recipe should probably simply disable the Windows shared configuration instead.

Reply:
+1 @toge